etc/inc/firefox-common-addons.inc

```diff
@@ -17,6 +17,7 @@ noblacklist ${HOME}/.kde4/share/config/kgetrc
 noblacklist ${HOME}/.kde4/share/config/okularpartrc
 noblacklist ${HOME}/.kde4/share/config/okularrc
 noblacklist ${HOME}/.local/share/kget
+noblacklist ${HOME}/.local/share/kxmlgui5/okular
```

Comment:
There is no `blacklist ${HOME}/.local/share/kxmlgui5/okular`.

Reply:
Should I add it to `disable-programs.inc` or remove this line?

Reply:
The `.local/share/kxmlgui` folder actually contains configuration files for quite a few KDE programs. Should I remove this line from this pull request, and then make a separate one which blacklists `.local/share/kxmlgui` in `disable-programs.inc` and then adds `noblacklist ${HOME}/.local/share/kxmlgui/application_name` in the profiles for the relevant KDE applications?

Reply:
I've removed it for now and will be making a separate pull request soon.